netlify.toml

```
@@ -2,6 +2,12 @@
   command = "yarn netlify-export"
   publish = "out"
 
+[[plugins]]
+  package = "@netlify/plugin-lighthouse"
+
```

Based on [this build results](https://app.netlify.com/sites/wargabantuwarga/deploys/60e5a1952bfca20007e095f2?utm_source=github&utm_campaign=bot_dl), let's set them as a threshold to keep the scores from degrading.

```suggestion

  [plugins.inputs.thresholds]
    performance = 0.91
    accessibility = 1
    best-practices = 1
    seo = 0.92
    pwa = 0.3

```

It's aligned with our [Avoid Unmeasured Improvements](https://github.com/kawalcovid19/wargabantuwarga.com#-unmeasured-improvements) principle.

---

Once this PR is merged with the above suggestion, we can start looking at #19 and measure the impacts of those improvements. 😁